app/controllers/users.js

```diff
@@ -43,19 +43,12 @@ exports.signIn = (req, res, next) => {
 };
 
 exports.listUsers = (req, res, next) => {
-  const { token, offset, limit } = req.query;
-  return validateToken(token)
-    .then(validated => {
-      if (validated) {
-        logger.info('User validated.\nGetting all users for listing them');
-        User.getUsers(offset, limit)
-          .then(foundUsers => {
-            res.status(200).send(foundUsers);
-          })
-          .catch(next);
-      } else {
-        throw validationError("User doesn't have permissions to request users list");
-      }
+  const { page, limit } = req.body,
+    offset = limit * (page - 1);
+
+  return User.getUsers(offset, limit)
```

Comment:
Did not notice this before, but you are missing a logger here. You can change it next PR.